basyx.components/basyx.components.docker/basyx.components.AASServer/src/main/java/org/eclipse/basyx/components/aas/AASServerComponent.java

```diff
@@ -150,7 +151,8 @@ public AASServerComponent(BaSyxContextConfiguration contextConfig, BaSyxAASServe
 	/**
 	 * Constructs an empty AAS server using the passed configuration
 	 */
-	public AASServerComponent(BaSyxContextConfiguration contextConfig, BaSyxAASServerConfiguration aasConfig, BaSyxMongoDBConfiguration mongoDBConfig) {
+	public AASServerComponent(BaSyxContextConfiguration contextConfig, BaSyxAASServerConfiguration aasConfig,
```

Comment:
There are lots of formatting changes unrelated to the actual change (integrating the new feature). Please revert the formatting changes